sample-configs/subdir/foo.conf

```diff
@@ -1,5 +1,5 @@
 foo=42
 # included without file()
 include "bar.conf"
-# included using file()
+# included using file() #### So far no file include support
```

Comment:
this should not have been commented out (my bad)
since we are expecting an error in test, maybe we can revert to the original content.

Reply:
`# included using file()` is a comment. `include file("bar-file.conf")` on line 5 is not commented out.